scripts/chue.coffee

```diff
@@ -57,10 +55,10 @@ module.exports = (robot) ->
               msg.emote "Sorry... " + body
 
   robot.respond /chue colou?r (\d+)? ?#?(.*)/i, (msg) ->
-    lamp = if msg.match[1] is undefined then "all" else msg.match[1]
+    lamp = if msg.match[1] is undefined then "" else msg.match[1]
```

Comment:
This might be a little confusing, as the florrie command now differs from the actual API call (i.e. lamp and colour are switched)... Not really a problem though.